README.md

```diff
@@ -1,19 +1,14 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Portfolio
+> Practice project developing a personal portfolio in various languages and frameworks.
 
 
 ## Built With
 
-- Major languages
+- HTML, CSS
 - Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+- Technologies
 
 
 ## Getting Started
```

Comment:
[OPTIONAL] For this section I recommend you add instructions on how to get a local copy of this project and how to install the necessary dependencies for it to run properly. It might seem like a trivial suggestion at the moment, but when you build more complex projects it will be a mandatory requirement, so it is better to get in the habit of doing so from early on.